hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/HandlerUtility.java

```diff
@@ -280,4 +284,35 @@ private static long calculateExpiration(
             return currentPlusMaxLife.getEpochSecond();
         }
     }
+
+    static void filterSignatoriesToRequired(Set<Key> signatories, Set<Key> required) {
+        final Set<Key> incomingSignatories = Set.copyOf(signatories);
+        signatories.clear();
+        filterSignatoriesToRequired(signatories, required, incomingSignatories);
+    }
+
+    private static void filterSignatoriesToRequired(
```

Comment:
It would be good to add javadoc for each @param.

Reply:
We generally don't javadoc private methods.
Is it needed for this to be understandable?  I can add it, but it likely won't say much more than the parameter name does...